doc/rados/operations/control.rst

```diff
@@ -329,7 +329,7 @@ The ``quorum`` list at the end lists monitor nodes that are part of the current
 
 This is also available more directly::
 
-	$ ./ceph quorum_status
+	ceph quorum_status
```

Comment:
but this isn't consistent with other commands, you see there are below 3 commands, all of them without `$`

![ceph-mon-stat](https://cloud.githubusercontent.com/assets/689798/17618550/fbc56902-60b2-11e6-8a58-f5f3dae253a6.png)

![ceph-mon_status](https://cloud.githubusercontent.com/assets/689798/17618559/02a6188e-60b3-11e6-96a7-b1a4f493851c.png)

![ceph-mon-dump](https://cloud.githubusercontent.com/assets/689798/17618602/4716c95a-60b3-11e6-80bb-221fff68cc00.png)